packages/ui/src/mocks/data/raw/applications.json

```diff
@@ -6,7 +6,16 @@
     "roleAccount": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
     "rewardAccount": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
     "stakingAccount": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
-    "answers": [],
+    "answers": [
+      {
+        "questionId": "0",
+        "answer": "18"
+      },
+      {
+        "questionId": "1",
+        "answer": "I am the smartest"
+      }
+    ],
```

Comment:
Let's review it after the schema upgrade. I worry that the mapping there is different but let's keep that in mind.

Reply:
Or event better -> please create a follow-up to check this and assign yourself to it :)